pkg/guid/guid.go

```diff
@@ -0,0 +1,32 @@
+package guid
+
+import (
+	"fmt"
+	"net"
+)
+
+// GUID address is an net.HardwareAddr encapsulation
+type GUID net.HardwareAddr
```

Comment:
i see alot of conversions to Uint64() in guid pool,

perhaps the type should  be `uint64` and the package would have a `func FromHardwareAddress(address net.HardwareAddr) GUID` ?

also add `GUID.HardwareAddress()` method for the other way around to be used in the daemon